core/src/main/kotlin/com/gitlab/kordlib/core/entity/Invite.kt

```diff
@@ -108,22 +105,23 @@ data class Invite(
      * @throws [RequestException] if anything went wrong during the request.
      * @throws [EntityNotFoundException] if the [Guild] wasn't present.
      */
-    suspend fun getGuild(): Guild = supplier.getGuild(guildId)
+    suspend fun getGuild(): Guild = supplier.getGuild(guildId!!)
```

Comment:
I shouldn't this become nullable then? Generally when an id is nullable the `getEntity` variant also is nullable

Reply:
I'll work on this in my next branch regarding the null assertation.

Reply:
In that case this should throw an `EntityNotFoundException` instead of a KNP

Reply:
There are various functions that return KNP instead of `EntityNotFoundException`. I'm going to work on those on a separate branch.